aloha-core/src/main/scala/com/eharmony/aloha/factory/ModelParser.scala

```diff
@@ -47,6 +47,12 @@ trait ModelParser extends JsValuePimpz {
     final def getParser[A: RefInfo, B: RefInfo: JsonReader: ScoreConverter](factory: ModelFactory, semantics: Option[Semantics[A]]) = new Parser[JsValue, Model[A, B]] {
         def parse(json: JsValue): Model[A, B] = json.convertTo(modelJsonReader[A, B](factory, semantics))
     }
+
+    // This is a very slightly modified copy of the lift from Additional formats that removes the type bound.
+    protected[this] def lift[A](reader: JsonReader[A]) = new JsonFormat[A] {
```

Comment:
This was extracted to the wrong place.  It should go in `com.eharmony.aloha.factory.ScalaJsonFormats` and should be implicit:

```
implicit def lift[A](implicit reader: JsonReader[A]) = new JsonFormat[A] {
  def write(a: A): JsValue = throw new UnsupportedOperationException("No JsonWriter[" + a.getClass + "] available")
  def read(value: JsValue) = reader.read(value)
}
```

Then we could change the next few lines.

```
import com.eharmony.aloha.factory.ScalaJsonFormats.lift // at top of file
val ast = json.convertTo[Ast[B]]
```